### PR TITLE
💄 Support dynamically generated og image feature

### PR DESCRIPTION
**Pull request by WeiJun0827**

### CODE REVIEW

**Review by williamchong — APPROVED** [review]

File: src/pages/_id/civic/index.vue [hunk 1]
```diff
@@ -147,7 +147,7 @@ export default {
         {
           hid: 'og:image',
           property: 'og:image',
-          content: 'https://liker.land/images/og/civic-v2.png',
+          content: `https://static.like.co/liker-og-image/${this.creator.user}`,
```
Comment (nwingt) [inline src/pages/_id/civic/index.vue:150]: Maybe refactor this line into a function?

**Review by nwingt — CHANGES_REQUESTED** [review]

File: src/pages/_id/civic/index.vue (hunk 1 above)
Comment (WeiJun0827) [reply src/pages/_id/civic/index.vue]: Updated, please check.

File: src/util/api/index.js [hunk 2]
```diff
@@ -131,3 +131,5 @@ export const getArticleDetailAPI = url =>
 
 export const updateProfile = () => `/api/users/self/update`;
 export const userPreferences = () => `/api/users/preferences`;
+
+export const getOgImage = id => `https://static.like.co/liker-og-image/${id}`;
```
Comment (nwingt) [inline src/util/api/index.js:135]: Suggest to rename this func for more specific use case
```suggestion
export const getLikerOgImage = id => `https://static.like.co/liker-og-image/${id}`;
```

**Review by nwingt — CHANGES_REQUESTED** [review]

File: src/util/api/index.js (hunk 2 above)
Comment (WeiJun0827) [reply src/util/api/index.js]: OK! Done~

**Review by nwingt — APPROVED** [review]

**Review by nwingt — APPROVED** [review]

**Pull request merged**